doc/content/integrations/storage/configuration.md

```diff
@@ -25,6 +25,16 @@ Initialize the configured database with:
 $ ttn-lw-stack storage-db init
 ```
 
+{{< note >}}
+If you are using TimescaleDB, the following configuration options are also available:
+
+```
+--timescaledb.chunk-time-interval
```

Comment:
This is used to configured the chunk time interval in the TimescaleDB hypertable. Perhaps good to link to the TimescaleDB documentation for this. https://docs.timescale.com/timescaledb/latest/how-to-guides/hypertables/best-practices/#time-intervals

The default value is set to 1 week, and people will typically not need to change this unless they really know what they are doing.

Reply:
Note that this is a duration, so it needs to be set as `168h` (168 hours) etc, or in general `Xh` for X hours.